common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import YAML from "yaml";
+
+import prettier from "prettier";
+
+import { AzSdkMetaTags } from "../commands/samples/publish";
+
+/**
+ * Options to control the generation of the sample package README.
+ */
+interface SampleReadmeConfiguration {
```

Comment:
this interface isn't exported, even though it's an input parameter to the default export?

Reply:
Created a little utility module called `sampleGenerationInfo.ts` to unify these.